projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader-config.types.ts

```diff
@@ -0,0 +1,24 @@
+import { InjectionToken } from '@angular/core';
+
+export interface NgxSkeletonLoaderConfig {
+  appearance?: 'circle' | 'line' | '';
+  animation?: 'progress' | 'progress-dark' | 'pulse' | 'false' | false;
+  theme?: {
+    // This is required since ngStyle is using `any` as well
+    // More details in https://angular.io/api/common/NgStyle
+    // tslint:disable-next-line: no-any
+    [k: string]: any;
+  };
+  loadingText?: string;
+  count?: number;
+}
+
+export const DEFAULT_NGX_SKELETON_LOADER_CONFIG = {
+  appearance: 'line',
+  animation: 'progress',
+  theme: {},
+  loadingText: 'Loading...',
+  count: 1,
+} as NgxSkeletonLoaderConfig;
```

Comment:
```suggestion
export const DEFAULT_NGX_SKELETON_LOADER_CONFIG: NgxSkeletonLoaderConfig = {
  appearance: 'line',
  animation: 'progress',
  theme: {},
  loadingText: 'Loading...',
  count: 1,
};
```

Reply:
Not necessary since `DEFAULT_NGX_SKELETON_LOADER_CONFIG` will be removed